bitrise.yml

```diff
@@ -13,26 +13,37 @@ workflows:
     - script@1.1.3:
         title: Bootstrapping...
         inputs:
-        - content: RUNNING_ON_CI=1 DISABLE_CARTHAGE_CACHE_CONFIG=true FORCE_CARTHAGE_VERSION=true
-            FORCE_SWIFTLINT_VERSION=true VERBOSE=true script/bootstrap
+        - content: |-
+            #!/bin/bash
+            set -ex
+
+            RUNNING_ON_CI=1 DISABLE_CARTHAGE_CACHE_CONFIG=true FORCE_CARTHAGE_VERSION=true FORCE_SWIFTLINT_VERSION=true VERBOSE=true script/bootstrap
     - script@1.1.3:
-        run_if: '{{getenv "PR" | eq "true"}}'
         title: Running linter...
         inputs:
-        - content:
+        - content: |-
             #!/bin/bash
             set -ex
-            echo `swiftlint lint --reporter json > .swiftlint-report.json || true`
-            bundle exec linterbot "$BITRISE_APP_SLUG" "$BITRISE_PULL_REQUEST" --trace < .swiftlint-report.json
+
+            mkdir swiftlint-output
+            swiftlint lint --reporter html > swiftlint-output/index.html || true
     - script@1.1.3:
         title: Testing...
         inputs:
-        - content: fastlane scan --scheme "Networking"
+        - content: bundle exec fastlane scan --scheme "Networking"
             COMMIT_MESSAGE:"$BITRISE_GIT_MESSAGE" REPO_ACCESS_TOKEN:"$REPO_ACCESS_TOKEN"
```

Comment:
This could be in the top line I think. As in "Bootstrapping" that as it is one command it's only in one line.